portafly/src/components/data-list/BulkSelectorWidget.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react'
+import {
+  Dropdown,
+  DropdownToggle,
+  DropdownToggleCheckbox,
+  DropdownItem
+} from '@patternfly/react-core'
+import { useTranslation } from 'i18n/useTranslation'
+import { useDataListTable, useDataListPagination } from 'components/data-list'
+
+interface Props {
+  filteredRows: any[]
+}
+
+const BulkSelectorWidget: React.FunctionComponent<Props> = ({
+  filteredRows
+}) => {
+  const { t } = useTranslation('accounts')
+  const { selectPage, selectAll, selectedRows } = useDataListTable()
+
+  const [isOpen, setIsOpen] = useState(false)
+
+  const selectedCount = selectedRows.length
+
+  const { startIdx, endIdx } = useDataListPagination()
+  const visibleRows = filteredRows.slice(startIdx, endIdx)
+  const pageCount = visibleRows.length
+  const allCount = filteredRows.length
+
+  const onSelect = () => {
+    setIsOpen((current) => !current)
+  }
+
+  // FIXME: Setting isChecked null creates an error in the console for uncontrolled React prop
```

Comment:
Is it mandatory that the prop of `DropdownToggleCheckbox` have 3 possible values? (true, false and null)

Reply:
What is puzzling is the fact that something called `isChecked` that suggest boolean type, can have a third one...  I know it's from PF, but something like `checkStatus` might've been more adequate.

Reply:
> What is puzzling is the fact that something called isChecked that suggest boolean type
Indeed. I would have used a prop `indeterminate` instead.

We have to live with this I'm afraid.